docs/build/specify-observables-pauli.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Specify observables in the Pauli basis
+description: Measure circuits in different Pauli bases, which is required to measure observables that are not diagonal in the computational basis.
+
+---
+
+# Specify observables in the Pauli basis
+
+In quantum mechanics, observables correspond to physical properties that can be measured.
+If we are dealing with system of spins, for example, we could be interested in measuring the system's energy or obtaining information about the alignment of the spins, such as the magnetization or the correlations between spins.
+
+To measure an $n$-qubit observable $O$ on a quantum computer, we have to represent it as sum of tensor products of Pauli operators, that is
+
+$$
+O = \sum_{k=1}^K \alpha_k P_k,~~ P_k \in \{I, X, Y, Z\}^{\otimes n},~~ \alpha_k \in \mathbb{R},
+$$
+
+where 
+
+$$
+I = \begin{pmatrix}
+1 & 0 \\ 0 & 1
+\end{pmatrix}
+~~
+X = \begin{pmatrix}
+0 & 1 \\ 1 & 0
+\end{pmatrix}
+~~
+Y = \begin{pmatrix}
+0 & -i \\ i & 0
+\end{pmatrix}
+~~
+Z = \begin{pmatrix}
+1 & 0 \\ 0 & -1
+\end{pmatrix}
+$$
+
+and we use the fact that an observable is hermitian, i.e. $O^\dagger = O$. If $O$ is not hermitian it can still be decomposed as sum of Paulis, but the coefficient $\alpha_k$ become complex.
+
+In many cases, the observable is naturally specified in this representation after mapping the system of interest to qubits.
+For example, a spin-1/2 system can be mapped to an Ising Hamiltonian
+
+$$
+H = \sum_{\langle i, j\rangle} Z_i Z_j + \sum_{i=1}^n X_i,
+$$
+
+where the indices $\langle i, j\rangle$ run over interacting spins and the spins are subject to a transversal field in $X$.
+The subscript index indicates which qubit the Pauli operator acts on, i.e. $X_i$ acts with an $X$ on qubit $i$ and leaves the rest unchanged.
+If we would like to measure the energy the observable is the Hamiltonian itself. Alternatively, we could measure the 
+average magnetization by counting the number of spins aligned in the $Z$-direction by
+
+$$
+O = \frac{1}{n} \sum_{i=1} Z_i
+$$
+
+If, instead, the observable is given in a matrix form, it can be expressed as Paulis by using the fact that the Pauli terms form a basis for the complex $2^n \times 2^n$ matrices.
+We expand the observable $O$ as 
+
+$$
+O = \sum_{P \in \{I, X, Y, Z\}^{\otimes n}} \mathrm{Tr}(O P) P,
+$$
+
+where the sum runs over all possible $n$-qubit Pauli terms and $\mathrm{Tr}(\cdot)$ is the trace of a matrix, which acts as inner product.
```

Comment:
```suggestion
The subscript index indicates which qubit the Pauli operator acts on. $X_i$ acts with an $X$ on qubit $i$ and leaves the rest unchanged.
If you want to measure the energy, the observable is the Hamiltonian itself. Alternatively, you could measure the 
average magnetization by counting the number of spins aligned in the $Z$-direction by

$$
O = \frac{1}{n} \sum_{i=1} Z_i
$$

If, instead, the observable is given in a matrix form, it can be expressed as Paulis by using the fact that the Pauli terms form a basis for the complex $2^n \times 2^n$ matrices.
We expand the observable $O$ as 

$$
O = \sum_{P \in \{I, X, Y, Z\}^{\otimes n}} \mathrm{Tr}(O P) P,
$$

where the sum runs over all possible $n$-qubit Pauli terms and $\mathrm{Tr}(\cdot)$ is the trace of a matrix, which acts as an inner product.
```